tutorials/tutorial_1/containers/data-prep/dataprep_gc.py

```diff
@@ -18,10 +18,9 @@ def convert_dcm_image_to_jpg(name):
 
 def dataset_dcm_to_jpg(dataset_df):
 	input_dir = '/input/dicom_data/'
-	output_dir = '/output/file_data/'
+	output_dir = '/output/0/file_data/'
```

Comment:
I don't think that this is required - users can still write to /output/file_data if they have a single output dataset.

Reply:
The error occurred while removing it:
```
Failed to import output datasets.Dataset 'Site 1 Dataset (v=3)': Failed to import output datasets. Error in prevalidation for the following fields:
JPG file: Field of type Filename requires file base path to be provided - please re-import and specify a file base path.
```